Drop -v linker flag, because it breaks the EB rpath wrapper.
author: micketeer@gmail.com
--- src/CMakeLists.txt.orig	2026-08-31 14:22:11.029170112 +0200
+++ src/CMakeLists.txt	2026-08-31 14:23:37.571832442 +0200
@@ -298,7 +298,6 @@
 )
 # configure no longer has access to any variable set below this line
 
-target_link_options(aflow PRIVATE "LINKER:-v")
 target_include_directories(aflow
         PRIVATE
         ${CMAKE_SOURCE_DIR}
